scss/forms/_meter.scss

```diff
@@ -37,7 +37,13 @@ $meter-fill-bad: $alert-color !default;
     height: $meter-height;
     margin-bottom: 1rem;
 
-    appearance: none;
+    // Disable `-webkit-appearance: none` from getting prefixed,
+    // We have disabled autoprefixer first and are just only using 
+    // `-moz-appearance: none` as a prefix and neglecting the webkit.
+    
+    /*! autoprefixer: off */
```

Comment:
Does this impact the rest of the rules for meter following this? (e.g. border-radius) down below... Do we need to separate out the appearance into a separately scoped `meter` rule?